wren-ai-service/docs/config_examples/config.google_ai_studio.yaml

```
@@ -16,8 +17,9 @@ type: embedder
 provider: litellm_embedder
 models:
   # put GEMINI_API_KEY=<your_api_key> in ~/.wrenai/.env
-  - model: gemini/text-embedding-004  # gemini/<gemini_model_name>
-    api_base: https://generativelanguage.googleapis.com/v1beta/openai  # change this according to your embedding model
+  - model: text-embedding-004
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 0 but found 2

(indentation)